### DISCUSSION TITLE
Symfony mailer RoudRobin "All transports failed" errors details

### DISCUSSION THREAD

**Discussion by sdespont** [symfony/symfony#57207] [Q&A] [answered]
I'm using Symfony Mailer 5.4.x with a RoundRobin configuration. 

Sometimes, the “All transports failed” error is thrown and I don't know why: the from/to email addresses exist and all the email accounts in the RoundRobin configuration are correct.

After checking the logs, there are no further details on the sending errors.

Is there any way of finding out the reasons for the exceptions for all attempts?

**Comment by MatTheCat** [comment, marked as answer]
The `TransportException` thrown by the `RoundRobinTransport` has a `$debug` property which aggregates transports debug info. You can access it through its `getDebug` method: https://github.com/symfony/symfony/blob/8ed9bb0b3ee0bed16b0dd885b0638f4e5a839d55/src/Symfony/Component/Mailer/Exception/TransportException.php#L19-L24

**Reply by sdespont** [reply]
Great thanks !
